test/feature/reverse_routing.feature

```diff
@@ -0,0 +1,31 @@
+Feature: reverse routing
+
+  Background:
+    Given an empty "controllers" directory
+
+
+  Scenario: routes can define reverse routes with 'as'
+    Given a file "routes.coffee" with the contents
+      """
+      module.exports = ({ GET }) ->
```

Comment:
Looking at this spec now, if I don't provide `as:`, then it should create a routing helper called `res.locals.paths.some_route`.